frontend/packages/console-shared/src/utils/owner-references.ts

```diff
@@ -16,7 +16,7 @@ export const compareOwnerReference = (
 
   return (
     obj.apiVersion === otherObj.apiVersion &&
-    obj.kind === otherObj.kind &&
+    kindForReference(obj.kind) === kindForReference(otherObj.kind) &&
```

Comment:
I'm confused why this change was needed. `obj.kind` should always be the k8s kind and not the internal console reference using `~`.

Reply:
from what I tested there are usages where the kind does contain `~`
it is a needed safety net that doesn't hurt

Reply:
What were the usages? That sounds like a bug.